src/Module/Smilies.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Friendica\Module;
+
+use Friendica\BaseModule;
+use Friendica\Core\Renderer;
+use Friendica\Core\System;
+
+/**
+ * Prints the possible Smilies of this node
+ */
+class Smilies extends BaseModule
+{
+	public static function content()
```

Comment:
Can you please move `content` after `rawContent`? In the App `rawContent` is called first.